hydromt_fiat/fiat.py

```diff
@@ -371,12 +377,15 @@ def setup_exposure_buildings(
         # In case the unit is passed as a pydantic value get the string
         if hasattr(unit, "value"):
             unit = unit.value
+
         self.exposure = ExposureVector(
             self.data_catalog,
             self.logger,
             self.region,
             unit=unit,
             damage_unit=damage_unit,
+            gfh_unit = gfh_unit,
```

Comment:
I don't understand why this object should have the arguments "gfh_unit" and "ground_elevation_unit". unit should be enough since that describes the units of the exposure object which should be uniform. The individual units should only be specified in the methods that set up the ground elevation and ground floor height from another source